CONTRIBUTING.md

```diff
@@ -0,0 +1,273 @@
+│   │   └── vite.config.ts  # Vite configuration
+│   ├── src/                # Source code for the binding
+│   │   ├── GlueTemplate.fs # Main binding code (renamed on generation)
+│   │   └── GlueTemplate.fsproj # Project file for the binding
+│   ├── .template.config/   # Template configuration
+│   │   └── template.json   # Template settings and post-actions
+│   ├── build.cmd           # Windows build script
+│   ├── build.sh            # Unix build script
+│   ├── Directory.*.props   # MSBuild property files
+│   └── ...                 # Other configuration files
+├── Glutinum.Template.proj  # Template packaging definition
+├── build.cmd               # Build script (Windows)
+├── build.sh                # Build script (Unix)
+├── README.md               # Main documentation
+└── ...                     # Other files
+```
+
+## MSBuild Property Files
+
+The template uses three important MSBuild property files to control build behavior and project configuration:
+
+### Directory.UserConfig.props
+
+This file is meant to be customized by the end user of the template. It contains project-specific settings:
+
+```xml
+<Project>
+    <PropertyGroup>
+        <!-- Package metadata -->
+        <Authors>Your Name</Authors>
+        <PackageProjectUrl>https://github.com/your-username/your-project</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/your-username/your-project.git</RepositoryUrl>
+        
+        <!-- Fable specific -->
+        <FablePackageType>binding</FablePackageType>
+        <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+        
+        <!-- Demo configuration -->
+        <GitHubDeployment>true</GitHubDeployment>
+    </PropertyGroup>
+</Project>
+```
+
+**Key Properties:**
+- `Authors`: Package author name(s)
+- `PackageProjectUrl`: URL to the project website/repository
+- `RepositoryUrl`: Git repository URL
+- `FablePackageType`: Sets the Fable package type (important for registry listing)
+- `GitHubDeployment`: Enables GitHub Pages deployment for the demo
+
+### Directory.Packages.props
+
+This file implements central package version management, allowing all package versions to be specified in one place:
+
+```xml
+<Project>
+    <ItemGroup>
+        <PackageVersion Include="Fable.Core" Version="4.0.0" />
+        <!-- Add other packages and their versions here -->
+    </ItemGroup>
+</Project>
+```
+
+**Key Concepts:**
+- Package versions are defined once in this file
+- Projects reference packages without specifying versions
+- Helps maintain consistent dependencies across projects
+- Updates to package versions only need to be made in one place
+
+### Directory.Build.props
+
+This file provides default MSBuild properties and behaviors that apply to all projects. Users should generally not modify this file unless they understand MSBuild well.
+
+It handles:
+- Default MSBuild settings
+- Fable configuration
+- Project conventions
+- Shared build properties
```

Comment:
I think this is already documented in the template manual.

Having it documented at a single place make it easier to keep in sync.

If needed, you can make a mention to read the template manual to learn more about it.